### ISSUE TITLE
How to quit broot safety?

### ISSUE THREAD

**Issue by Erimus-Koo** [Canop/broot#718]
<img width="1360" alt="image" src="https://github.com/Canop/broot/assets/17139670/a69a2a3b-8631-4540-8047-d2643ee0094e">

Sometimes broot still runs in the background and burns my computer.

Maybe because I ran broot start at $HOME path, and there were too many files need to be scanned, and `Ctrl+Q` hadn't broken all tasks?

**Comment by Canop** [comment]
Can you find out a reproduction sequence then execute it with [debug log enabled](https://dystroy.org/broot/community/#log) then post the log ?

**Comment by Erimus-Koo** [comment]
> Can you find out a reproduction sequence then execute it with [debug log enabled](https://dystroy.org/broot/community/#log) then post the log ?

I can have a try.

And I've opened the log and there is `broot.log` in every folder where I started broot, can I set a specific folder or file for the log file?

**Comment by Canop** [comment]
> And I've opened the log and there is broot.log in every folder where I started broot, can I set a specific folder or file for the log file?

Right now you can't set the log location but you're not supposed to log every time, only when reproducing a problem for analysis. Even me, I don't often enable this log.
If your shell allows it, you can run `BROOT_LOG=debug br` on the precise time you want the log to be enabled.

**Comment by Erimus-Koo** [comment]
I've tried, but I can't reproduce it every time.
I'll keep watching for a few days.

**Comment by Erimus-Koo** [comment]
<img width="740" alt="image" src="https://github.com/Canop/broot/assets/17139670/bfc510ed-f4ec-4835-8d29-096a7fb72c46">

It's a bit difficult to reproduce.

And a folder has been removed after Stage. I'm not sure if I staged too many files curse some problem, or selected that folder mistakenly.

Still not log file, sigh. I don't want the `broot.log` to appear in my folders randomly.

**Comment by Canop** [comment]
Dit it happen again ?

**Comment by Erimus-Koo** [comment]
Yeah, several times, but not so often, and the reason is also not clear.

**Comment by Canop** [comment]
You don't have to close the issue just because it's hard to reproduce. I'd rather have it stay open. Even if you don't manage to reproduce, others may have the problem too and may find the issue.

**Comment by Erimus-Koo** [comment]
Fine, I'll update if there's something new.

**Comment by BlueskyFR** [comment]
Maybe rename to "Broot still running in background" to get more attention?

**Comment by Canop** [comment]
Right. Done.
